packages/scene-composer/src/utils/entityModelUtils/cameraComponent.ts

```diff
@@ -0,0 +1,95 @@
+import { ComponentRequest, ComponentUpdateRequest } from '@aws-sdk/client-iottwinmaker';
+import { DocumentType } from '@aws-sdk/types';
+
+import { ICameraComponent, KnownComponentType } from '../../interfaces';
+import { componentTypeToId } from '../../common/entityModelConstants';
+import { ICameraComponentInternal } from '../../store';
+import { generateUUID } from '../mathUtils';
+import { CameraType } from '../../models/SceneModels';
+import { DEFAULT_CAMERA_OPTIONS } from '../../common/constants';
+
+enum CameraComponentProperty {
+  CameraType = 'cameraType',
+  Fov = 'fov',
+  Near = 'near',
+  Far = 'far',
+  Zoom = 'zoom',
+}
+
+export const createCameraEntityComponent = (camera: ICameraComponent): ComponentRequest => {
+  const comp: ComponentRequest = {
+    componentTypeId: componentTypeToId[KnownComponentType.Camera],
+    properties: {
+      [CameraComponentProperty.CameraType]: {
+        value: {
+          stringValue: camera.cameraType,
+        },
+      },
+    },
+  };
+
+  if (camera.fov) {
```

Comment:
Nit-pick: I think this would be a good scenario for using a `switch`, but that's just a personal preference.

Reply:
switch will not work. we want to assign all values that is available instead of handling different cases of a single value